sheets_and_friends/tsv_in/modifications_long.tsv

```diff
@@ -96,13 +96,13 @@ EmslInterface	emsl_store_temp	replace_attribute	required			true
 EmslInterface	emsl_store_temp	replace_attribute	recommended			false
 AirInterface|BiofilmInterface|BuiltEnvInterface|HcrCoresInterface|HcrFluidsSwabsInterface|HostAssociatedInterface|MiscEnvsInterface|PlantAssociatedInterface|SedimentInterface|SoilInterface|WastewaterSludgeInterface	env_broad_scale	replace_attribute	description			In this field, report which major environmental system your sample or specimen came from. The systems identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. were you in the desert or a rainforest?). We recommend using subclasses of ENVO's biome class: http://purl.obolibrary.org/obo/ENVO_00000428. Format (one term): termLabel [termID], Format (multiple terms): termLabel [termID]|termLabel [termID]|termLabel [termID]. Example: Annotating a water sample from the photic zone in middle of the Atlantic Ocean, consider: oceanic epipelagic zone biome [ENVO:01000033]. Example: Annotating a sample from the Amazon rainforest consider: tropical moist broadleaf forest biome [ENVO:01000228]. If needed, request new terms on the ENVO tracker, identified here: http://www.obofoundry.org/ontology/envo.html
 AirInterface|BiofilmInterface|BuiltEnvInterface|HcrCoresInterface|HcrFluidsSwabsInterface|HostAssociatedInterface|MiscEnvsInterface|PlantAssociatedInterface|SedimentInterface|SoilInterface|WastewaterSludgeInterface	env_broad_scale	add_attribute	notes			"range changed to enumeration late in makefile, so this is modified (but ""sample ID"" anyway)"
-SoilInterface	env_broad_scale	remove_attribute	range
-SoilInterface	env_broad_scale	replace_attribute	any_of.0.range			EnvBroadScaleSoilEnum
+SoilInterface	env_broad_scale	remove_attribute	range			
+SoilInterface	env_broad_scale	replace_attribute	any_of.0.range			"EnvBroadScaleSoilEnum  "
```

Comment:
This doesn't look right to me. I don't think there should be any trailing white-space after `EnvBroadScaleSoilEnum`. The quotation marks drew my eye to it. They should be removed to. They were probably added by your spreadsheet application. _Sometimes_ it's more productive to edit these files in a good plain text editor.

Reply:
Weird! I was using VSCode but I think an extension snuck in there to mess things up. Good catch thanks